zcash_client_sqlite/src/lib.rs

```diff
@@ -1267,6 +1316,12 @@ impl<P: consensus::Parameters> WalletWrite for WalletDb<rusqlite::Connection, P>
                         .enumerate()
                     {
                         if let Some(address) = txout.recipient_address() {
+                            // TODO: we really want to only mark outputs when a transaction has been
+                            // *reliably* mined, because that is strictly more conservative in avoiding
+                            // going over the gap limit.
```

Comment:
```suggestion
```
This is the wrong place for this comment. We have enough information in the database to consider the number of confirmations of the oldest tx that mines an output to this address if we want to. That would be done in `zcash_client_sqlite::wallet::transparent::ephemeral::first_unsafe_index`, so I will add a comment there instead.